src/Jobs.php

```diff
@@ -62,18 +63,25 @@ public static function create(
         $filename,
         $runsample = null,
         $autoparse = null,
-        $autostart = null
+        $autostart = null,
+        $addHistoricalData = null
```

Comment:
Let's rename this parameter to `historicalData` to keep it consistent with the other SDKs